instrumentation.go

```diff
@@ -316,6 +316,10 @@ var lookupEnv = os.LookupEnv
 // and [WithServiceName] if their respective environment variable is defined.
 // If more than one of these options are used, the last one provided to an
 // [Instrumentation] will be used.
+//
+// The OTEL_TRACES_EXPORTER environment variable value is resolve using the
+// [autoexport] package. See that package's documentation for information on
```

Comment:
```suggestion
// [go.opentelemetry.io/contrib/exporters/autoexport] package. See that package's documentation for information on
```

Reply:
This currently uses the Go doc linking. The proposed change will not.

Reply:
> When referring to other packages, “pkg” can be either a full import path or the assumed package name of an existing import.

I was not aware of it 👍